docs/guides/testing.md

```diff
@@ -84,7 +84,7 @@ it('makes sure a user was registered', async () => {
 
 ## With a Database
 
-Integration testing with a databsae can add a lot of complexity to your test suite. But Nexus is in a good position to help since it knows about both test and database domains of your app. The following assumes you are using a [db driver](/guides/databases?id=driver-system). It is _not_ about database testing _in general_.
```

Comment:
I'm not sure anymore what to use as word instead of db driver

Reply:
We just don't mention it now I guess. Remove the last two sentences?